host/sgx/switchless.c

```diff
@@ -37,6 +114,12 @@ static oe_result_t oe_stop_worker_threads(oe_switchless_call_manager_t* manager)
     for (size_t i = 0; i < manager->num_host_workers; i++)
     {
         manager->host_worker_contexts[i].is_stopping = true;
+        _worker_wake(&manager->host_worker_contexts[i]);
+
+        OE_TRACE_INFO(
+            "Switchless host worker thread %d spun for %lu times",
```

Comment:
%d does not match size_t. I wonder why the compiler did not catch this?

Reply:
Fixed.